### DISCUSSION TITLE
Addon Ideas #5

### DISCUSSION THREAD

**Discussion by SpryzenR** [ScratchAddons/ScratchAddons#6273] [Ideas]
### Hide Stage/Backdrops Sidebar Add-On:
Makes a button that allows the stage sidebar to be hidden and create more space for the sprites and its properties information.  

### Hide Parts of the Sprite Properties Add-On:

Makes buttons that hides the information of the different sprite properties (such as the direction and the x & y position of a sprite), giving more space for properties that are not hidden.

**Comment by DNin01** [comment]
By hiding the stage sidebar, do you mean something like #5218?

**Reply by mxmou** [reply]
The OP probably meant the stage selector:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/eda0ad2e-a6c4-434f-aeef-02a8fa2a2d22)

**Reply by DNin01** [reply]
Oh, I get it now - stage/backdrops.

**Reply by SpryzenR** [reply]
Sorry idk some of the names of the stuff on Scratch but, yes I mean that.
